otter/models/cass.py

```diff
@@ -1789,17 +1794,18 @@ def insert_servers(self, last_update, servers, clear_others):
             queries.append(query.format(cf=self.table, i=i))
         if clear_others:
             return self.delete_servers().on(
-                lambda _: cql_eff(batch(queries), params))
+                lambda _: cql_eff(
+                    batch(queries, self.clock.seconds()), params))
         else:
-            return cql_eff(batch(queries), params)
+            return cql_eff(batch(queries, self.clock.seconds()), params)
```

Comment:
Great catch @cyli. This should be microseconds. See [docs](http://docs.datastax.com/en/cql/3.0/cql/cql_reference/delete_r.html). Will change it.